torchtext/csrc/vocab.cpp

```diff
@@ -96,7 +96,18 @@ std::vector<int64_t> Vocab::lookup_indices(const StringList &tokens) {
   return indices;
 }
 
-c10::Dict<std::string, int64_t> Vocab::get_stoi() const { return stoi_; }
+std::unordered_map<std::string, int64_t> Vocab::get_stoi() const {
```

Comment:
Same here. We can limit the scope of the PR and add this change later for debug purpose.